component-definition.json

```diff
@@ -535,6 +535,38 @@
               }
             }
           }
+        },
+        {
+          "title": "Loan Steps",
+          "id": "loan-steps",
+          "plugins": {
+            "xwalk": {
+              "page": {
+                "resourceType": "core/franklin/components/block/v1/block",
+                "template": {
+                  "name": "Loan Steps",
+                  "filter": "loan-steps",
+                  "model": "loan-steps"
+                }
+              }
+            }
+          }
+        },
+        {
+          "title": "Loan Step",
+          "id": "loan-step",
+          "plugins": {
+            "xwalk": {
+              "page": {
+                "resourceType": "core/franklin/components/block/v1/block/item",
+                "template": {
+                  "name": "Loan Step",
+                  "model": "loan-step",
+                  "filter": "loan-step"
```

Comment:
there is no filter for loan-step, you anyway can not add more nested elements below it.